logging/unit_tests/handlers/test_container_engine.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+
+class TestContainerEngineHandler(unittest.TestCase):
+    PROJECT = 'PROJECT'
+
+    def _getTargetClass(self):
+        from google.cloud.logging.handlers.container_engine import (
+            ContainerEngineHandler)
+        return ContainerEngineHandler
+
+    def _makeOne(self, *args, **kw):
+        return self._getTargetClass()(*args, **kw)
+
+    def test_format(self):
+        import logging
+        import json
+        handler = self._makeOne()
+        LOGNAME = 'loggername'
+        MESSAGE = 'hello world'
+        record = _Record(LOGNAME, logging.INFO, MESSAGE)
+        expected_payload = {'message': MESSAGE,
+                            'timestamp': {'seconds': 5,
+                                          'nanos': int(.03 * 1e9)},
+                            'thread': record.thread,
+                            'severity': record.levelname}
```

Comment:
While you're in there adding trailing commas you may as well reformat this dict to look like the rest?